app/pulp/app/serializers/repository.py

```diff
@@ -15,11 +15,26 @@ class RepositorySerializer(ModelSerializer):
         help_text='A unique name for this repository.'
     )
 
+    description = serializers.CharField(
+        help_text='An optional description.',
+        required=False
+    )
+
+    last_content_added = serializers.DateTimeField(
+        help_text='When content was last added.',
```

Comment:
For some background, my thinking here is that the API is user-facing, so it might be worth being a little bit more verbose than what we've got in the Model docstrings due to the different audience (API user vs. Python developer).